src/index.tsx

```diff
@@ -13,13 +14,22 @@ WebFontLoader.load({
   }
 });
 
+const providerConfig = {
+  domain: 'dev-l0ct8qiv.eu.auth0.com',
+  clientId: 'cmEcZQZ8ENMuAR4tUZrAlJo6pzGaP5C1',
+  redirectUri: window.location.origin
+};
+
 ReactDOM.render(
-  <React.StrictMode>
-    <GlobalStyle />
-    <KwitterThemeProvider>
-      <App />
-    </KwitterThemeProvider>
-  </React.StrictMode>,
+  <Auth0Provider {...providerConfig}>
```

Comment:
Might be better to keep the StrictMode component to be at the top level? 🙂

Reply:
yeah you are right!